gradle-conjure/src/main/java/com/palantir/gradle/conjure/Dependencies.java

```diff
@@ -31,14 +34,36 @@ final class Dependencies {
     static final String CONJURE_JAVA_LIB = "com.palantir.conjure.java:conjure-lib:8.22.0";
     static final String CONJURE_UNDERTOW_LIB = "com.palantir.conjure.java:conjure-undertow-lib:8.22.0";
     static final String DIALOGUE_TARGET = "com.palantir.dialogue:dialogue-target:3.135.0";
+
+    static final String GUAVA = "com.google.guava:guava:33.4.8-jre";
+    static final String UNDERTOW_CORE = "io.undertow:undertow-core:2.3.18.Final";
+    static final String JACKSON_ANNOTATIONS = "com.fasterxml.jackson.core:jackson-annotations:2.18.3";
+    static final String JACKSON_DATABIND = "com.fasterxml.jackson.core:jackson-databind:2.18.3";
+    static final String FINDBUGS = "com.google.code.findbugs:jsr305:3.0.2";
+    static final String ERROR_PRONE_ANNOTATIONS = "com.google.errorprone:error_prone_annotations:2.38.0";
+    static final String SAFE_LOGGING_PRECONDITIONS = "com.palantir.safe-logging:preconditions:3.9.0";
+    static final String SAFE_LOGGING = "com.palantir.safe-logging:safe-logging:3.9.0";
```

Comment:
To check: we may also need a dependency on conjure-java-runtime-api somewhere to depend on RemoteException? (for dialogue-generated clients)